### ISSUE TITLE
Can't use it with `@web/dev-server`

### ISSUE THREAD

**Issue by abdonrd** [umbopepato/rollup-plugin-postcss-lit#45]
I just tried to use it with the `@web/dev-server`, but I get this error:

>Error: fromRollup should be called with a rollup plugin function. Received: [object Object]

The config:

```javascript
// ...
import { fromRollup } from '@web/dev-server-rollup';
import rollupPostcss from 'rollup-plugin-postcss';
import rollupPostcssLit from 'rollup-plugin-postcss-lit';

const postcss = fromRollup(rollupPostcss);
const postcssLit = fromRollup(rollupPostcssLit);

export default {
  // ...
  mimeTypes: {
    '**/*.scss': 'js',
  },
  plugins: [
    // ...
    postcss({
      inject: false,
    }),
    postcssLit({
      importPackage: 'lit',
    }),
  ],
};
```

**Comment by abdonrd** [comment]
Oh! It works change it from:

```javascript
const postcssLit = fromRollup(rollupPostcssLit);
```

To:

```javascript
const postcssLit = fromRollup(rollupPostcssLit.default);
```

**Comment by umbopepato** [comment]
Hey @abdonrd, thanks for reporting this!
I've switched to `export =` so that `.default` is not required anymore in node esm, the fix will be available in `v1.1.1` soon 🙂
